content/06.rule5.md

```diff
@@ -0,0 +1 @@
+## Rule 5: Tune your hyperparameters
```

Comment:
Could be extended to 

"Rule 5: Tune your hyperparameters extensively and systematically" -- the extensively, to distinguish this a little bit from applying off-the-shelf algorithms like random forests